panther_analysis_tool/main.py

```diff
@@ -80,12 +80,15 @@ def get(self, arg: str, default: Any = None) -> Any:
             str,
         Optional('Suppressions'): [str],
         Optional('Tags'): [str],
+        Optional('Reports'): {
```

Comment:
Can we also fix the PolicyID vs RuleID issues in the schema?

Reply:
ResourceType vs LogType

Reply:
That's really more of a backend Panther change than a change the `panther_analysis_tool` needs, the backend currently doesn't have a concept of RuleID or LogType so we could let the `panther_analysis_tool` support these fields, but anyone who actually used them wouldn't be able to upload those rules to Panther.